integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/datamap/CarbonCreateDataMapCommand.scala

```diff
@@ -50,13 +51,30 @@ case class CarbonCreateDataMapCommand(
     val LOGGER = LogServiceFactory.getLogService(this.getClass.getCanonicalName)
     if (dmClassName.equals("org.apache.carbondata.datamap.AggregateDataMapHandler") ||
         dmClassName.equalsIgnoreCase("preaggregate")) {
-      CreatePreAggregateTableCommand(
-        dataMapName,
-        tableIdentifier,
-        dmClassName,
-        dmproperties,
-        queryString.get
-      ).processMetadata(sparkSession)
+      val timeHierarchyString = dmproperties.get("timeseries.hierarchy")
```

Comment:
please collect all datamap property key string for pre-agg table and put them into a new file, so that it is easier to find all the properties for pre-agg table

Reply:
ok